hw4/02-sessions.js

```diff
@@ -1,7 +1,7 @@
 const express = require('express');
 const session = require('express-session');
 const app = express();
-const port = process.env.PORT || 5000;
+const port = process.env.PORT || 5001;
```

Comment:
you can merge upstream caterina changes in main and merge your main in this branch so that unnecessary changes are not reflected.

Reply:
Thank you!